x-pack/plugins/actions/server/saved_objects/migrations.ts

```diff
@@ -77,6 +76,26 @@ function renameCasesConfigurationObject(
   };
 }
 
+function removeCasesFieldMappings(
+  doc: SavedObjectUnsanitizedDoc<RawAction>
+): SavedObjectUnsanitizedDoc<RawAction> {
+  if (
+    !doc.attributes.config?.hasOwnProperty('isCaseOwned') &&
```

Comment:
I think is safer to do (replace `&&` with `||`). I am not sure if all savedObject will have both of the properties as `isCaseOwned` introduced after `incidentConfiguration`. Maybe there i a migration that ensures the existence of `isCaseOwned`. @YulNaumenko ?

```suggestion
    !doc.attributes.config?.hasOwnProperty('isCaseOwned') ||
    !doc.attributes.config?.hasOwnProperty('incidentConfiguration')
```

Reply:
if we put an `||`, it will be true if one of the params is there and the other is not, and it would not remove the param

Reply:
Oh yeah, you are right. What I meant is that we should enter the migration if one of them is presented. I misread it. Thanks!